OrbitGl/App.cpp

```diff
@@ -138,12 +138,12 @@ void OrbitApp::OnCaptureStarted(ProcessData&& process,
 
         // It is safe to do this write on the main thread, as the capture thread is suspended until
         // this task is completely executed.
-        capture_data_ =
-            CaptureData(std::move(process), module_manager_.get(), std::move(selected_functions),
-                        std::move(selected_tracepoints), std::move(user_defined_capture_data));
+        capture_data_ = CaptureData(std::move(process), module_manager_.get(),
+                                    std::move(selected_functions), std::move(selected_tracepoints));
 
-        frame_track_online_processor_ =
-            FrameTrackOnlineProcessor(GetCaptureData(), GCurrentTimeGraph);
+        data_manager_->set_user_defined_capture_data(std::move(user_defined_capture_data));
```

Comment:
It feels a bit strange that we set the user defined capture data here, which will be the same value as it is right now in data_manager.
But probably it is fine for now.